Content.Server/Disease/Events/AttemptSneezeCoughEvent.cs

```diff
@@ -0,0 +1,38 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Disease.Events;
+
+/// <summary>
+///     Raised by an entity about to sneeze/cough.
+/// </summary>
+//[ByRefEvent]
+public sealed class AttemptSneezeCoughEvent : EntityEventArgs
```

Comment:
Any particular reason this can't be a by-ref record struct?
```
[ByRefEvent]
public record struct AttemptSneezeCoughEvent(EntityUid uid, string SnoughMessage, SoundSpecifier? SnoughSound, bool Cancelled = false);
```

Significantly less stuff needs written out.

Reply:
Inexperience with C#... I definitely like how much more compact that is